docs/griptape-cloud/knowledge-bases/accessing-data.md

```diff
@@ -0,0 +1,33 @@
+# Accessing Data in a Knowledge Base
+
+You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.
```

Comment:
Can we add that Search is used in the GriptapeCloudKnowledgeBase Tool and Query is used in GriptapeCloudKnowledgeBaseVectorStoreDriver? May help users understand applications/use cases

Reply:
Ah yes, I'll call that out down in the framework section.